apps/src/templates/progress/ProgressLesson.jsx

```diff
@@ -151,7 +151,10 @@ class ProgressLesson extends React.Component {
         ? lesson.description_teacher
         : lesson.description_student;
 
-    const levelUrl = levels[0] && levels[0].url;
+    // There's no url for a lesson so use the url of the first level of the lesson
+    // as the url for the lesson.
+    // TODO: Make the back-end support lesson urls and redirect to the first level.
```

Comment:
Mind linking a task for this TODO in the Github PR comments?

Reply:
Sorry! I meant in this comment thread. (i.e. not in the code itself). It's not a hard and fast rule, but we generally avoid putting the task directly in the code. The main reason I've heard is since this is open source and we encourage people to fork for their own uses, having the task link in the code is potentially confusing for external groups.

Reply:
Oh oops, sorry I misread your comment.  Here's the link to the Jira task: https://codedotorg.atlassian.net/browse/LP-1634.